WooCommerce/Classes/ViewRelated/Products/ProductsViewController.swift

```diff
@@ -949,13 +949,13 @@ private extension ProductsViewController {
         case .noResultsPlaceholder:
             displayNoResultsOverlay()
         case .syncing(let pageNumber):
-            if pageNumber == SyncingCoordinator.Defaults.pageFirstIndex {
-                displayPlaceholderProducts()
```

Comment:
I think we might want to keep showing this `displayPlaceholderProducts` when syncing the first page? otherwise, if there are no results before (like switching to a different store and going to the products tab) the products tab appears empty:

<img src="https://user-images.githubusercontent.com/1945542/156516087-c617bdce-0adb-428f-b8ce-bf0af26f2988.png" width="300" />